owasp-suppression.xml

```diff
@@ -1257,4 +1257,12 @@
         <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
         <vulnerabilityName>CVE-2023-35116</vulnerabilityName>
     </suppress>
+<!--    This cve concerns https://github.com/billdavidson/JSONUtil/issues/10-->
```

Comment:
I would add some info why it's suppressed. Something similar to:
` </suppress>
	<!-- False positive, H2 web console is used in development process. Not shared in production -->
	<suppress>
   		<notes><![CDATA[
   		file name: h2-1.4.197.jar
   		]]></notes>
   		<packageUrl regex="true">^pkg:maven/com\.h2database/h2@.*$</packageUrl>
   		<cve>CVE-2022-45868</cve>
	</suppress>`